chart/values.yaml

```diff
@@ -113,6 +113,13 @@ secrets:
   spawningToken:
     fromSecret: true
     secretName: "spawning-token"
+  cachedAssetsS3:
+    accessKeyId:
+      fromSecret: true
+      secretName: "cached-assets-s3-access-key-id"
```

Comment:
are they created in our infra?

Reply:
I think so https://github.com/huggingface/infra/blob/8978f9832b5b8f2248bb228b6554f89c0af36a83/projects/hub/kubernetes_app_datasets_server.tf#L136